config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :create, :new] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] There should be a pair of routes in the _config/routes.rb_ file for creating _comments_. Kindly improve upon this aspect ensuring the _comments_ routes corresponds to:

```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
 GET   /posts/:post_id/comments/new(.:format)     comments#new
```

Reply:
done, thanks!

Reply:
done, thanks!